<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginCloseButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>0</y>
     <width>43</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="accountCheckBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>记住账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码(&amp;P)</string>
   </property>
   <property name="buddy">
    <cstring>passwordLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账号(&amp;A)</string>
   </property>
   <property name="buddy">
    <cstring>accountComboBox</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>loginButton</zorder>
  <zorder>loginCloseButton</zorder>
  <zorder>accountCheckBox</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>passwordLineEdit</zorder>
  <zorder>accountLabel</zorder>
  <zorder>accountComboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
